ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -129,10 +133,47 @@ Test Inference With Token Authentication
 
 Verify Multiple Projects With Same Model
     [Documentation]    Test the deployment of multiple DS project with same openvino_ir model
-    [Tags]    Sanity
-    ...    RHOAIENG-549    RHOAIENG-2724
+    [Tags]    Sanity    Tier1
+    ...       RHOAIENG-549    RHOAIENG-2724
     Create Openvino Models    server_name=${RUNTIME_NAME}    model_name=${MODEL_NAME}    project_name=${PRJ_TITLE}
-    ...    num_projects=5
+    ...    num_projects=4
+    [Teardown]    Run Keyword If Test Failed    Get Modelmesh Events And Logs
+    ...    server_name=${RUNTIME_NAME}    project_title=${PRJ_TITLE}
+
+Verify Editing Existing Model Deployment
+    [Documentation]    Tries editing an existing model deployment to see if the underlying deployment is updated
+    [Tags]    Sanity    Tier1
+    ...       ProductBug    RHOAIENG-2869
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
```

Comment:
To regress bug RHOAIENG-2869 we need to update existing model, which is tied to an existing DS project. 

i.e. we need to call this KW with `existing_project=${TRUE}` (and only after project `${PRJ_TITLE}` already exists, with same model name `${MODEL_NAME}`.

```suggestion
    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}    existing_project=${TRUE}
```

Reply:
This is done in lines 166 to 168, where we try editing the existing model and querying it; I've done it this way in order to decouple this test from other ones, otherwise we'd always need to expect a DS Project to already exist with a given name and deployed model

Reply:
Although in line 168 you use `existing_model=${TRUE}`, it will still be a different instance, even if the model name is the same. 
This is because we can have same model name across different DS projects, so it's different model instance.
To edit existing model, you must edit **_existing model in an existing DS project_**.

BTW,  see the new test `Verify Multiple Projects With Same Model` where we use the same model name, but in different project names, so we get multiple models with same name across different projects.

Reply:
Oh I think I missed line 153, where you already created that model name in the project...
In that case, we're all good :)